### PR TITLE
Add examples/play-from-disk

### PR DESCRIPTION
**Pull request by Sean-Der**
Using IVFReader demonstrate how user can play videos from hard disk to browser

### CODE REVIEW

File: examples/play-from-disk/main.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	}
+
+	// Create a new RTCPeerConnection
+	peerConnection, err := webrtc.NewPeerConnection(config)
+	if err != nil {
+		panic(err)
+	}
+
+	// Create a video track
+	videoTrack, err := peerConnection.NewTrack(webrtc.DefaultPayloadTypeVP8, rand.Uint32(), "video", "pion")
```
Comment (fivebats) [inline examples/play-from-disk/main.go]: That (webrtc.DefaultPayloadTypeVP8) is not going to work on Safari or recent Firefox. You must use the dynamic payload type for VP8 chosen by the offering browser. See what I did in https://github.com/pion/example-webrtc-applications/pull/22  (using firstCodecOfType) to get the VP8 payload type

File: examples/play-from-disk/jsfiddle/demo.details [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+ name: gstreamer-send
+ description: Example of using pion-WebRTC to send video to your browser using GStreamer
```
Comment (rviscarra) [inline examples/play-from-disk/jsfiddle/demo.details]: This needs to be updated.

File: examples/play-from-disk/jsfiddle/demo.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+pc.oniceconnectionstatechange = e => log(pc.iceConnectionState)
+pc.onicecandidate = event => {
+  if (event.candidate === null) {
+    document.getElementById('localSessionDescription').value = btoa(JSON.stringify(pc.localDescription))
+  }
+}
+
+// Offer to receive 1 audio, and 2 video tracks
+pc.addTransceiver('audio', {'direction': 'sendrecv'})
+pc.addTransceiver('video', {'direction': 'sendrecv'})
```
Comment (rviscarra) [inline examples/play-from-disk/jsfiddle/demo.js]: Remove the extra audio and video transceivers.

Comment (Sean-Der) [reply examples/play-from-disk/jsfiddle/demo.js]: Fixed!

File: examples/play-from-disk/jsfiddle/demo.details (hunk 2 above)
Comment (Sean-Der) [reply examples/play-from-disk/jsfiddle/demo.details]: Fixed!

**Pull request merged**